moabb/pipelines/utils.py

```diff
@@ -45,6 +45,7 @@ def create_pipeline_from_config(config):
             params = component["parameters"]
             if "optimizer" in component["parameters"].keys():
                 for optm in component["parameters"]["optimizer"]:
+                    print(optm)
```

Comment:
I temporarily inserted the print to discuss and decide something. 

We add a special parser for the callback and the optimizer with TensorFlow. 

Now, we need something similar to the braindecode object. The problem is establishing some difference between the braindecode and TensorFlow YAMLs file since both share the same name for callbacks, and optimizer, among other variables.

My ideas/solutions so far,

1) we add some flag into all the YAML files with deep learning, something like: "engine: braindecode" or  "engine: Tensorflow".

2) we first try the one, and if we fail, we test the other (will always fail if it is the wrong signature because the parameters are always diff because of the silly rivalry between tensorflow and torch). 

3) We could try to manage a way to build a parse that could be used for both. 

I would like some assistance in deciding which path I follow, @carraraig and @sylvchev

Reply:
Option 1 :)

Reply:
I think we could use option 1, it seems better to expose the engine directly in the pipeline.